src/models/stsApi.ts

```diff
@@ -104,10 +104,12 @@ export async function getContextPath(processKey: string) {
  */
 export async function getUrlOfBeanType(beanType: string) {
     const bindingKey = `L${beanType.replace(/\./g, "/")};`;
-    const uriString = await vscode.commands.executeCommand<string>("sts.java.javadocHoverLink", {
-        bindingKey,
-        lookInOtherProjects: true
-    });
+    const uriString = await vscode.commands.executeCommand<string>("java.execute.workspaceCommand",
```

Comment:
here wrap it with `java.execute.workspaceCommand`, looks like the latest redhat.java doesn't register vscode commands for delegate commands.